### ISSUE TITLE
Fatal error in , line 0 Check failed: has_pending_exception().

### ISSUE THREAD

**Issue by wuast94** [node-red/node-red#4304]
### Current Behavior

my nodered runs on the office docker container and restarts because of an error:

```
#
# Fatal error in , line 0
# Check failed: has_pending_exception().
#
#
#
#FailureMessage Object: 0x7ffcd28a4440
 1: 0xb72001  [node-red]
 2: 0x1bfa314 V8_Fatal(char const*, ...) [node-red]
 3: 0xe22521 v8::internal::Isolate::UnwindAndFindHandler() [node-red]
 4: 0x11e513c v8::internal::Runtime_UnwindAndFindExceptionHandler(int, unsigned long*, v8::internal::Isolate*) [node-red]
 5: 0x15d9f95  [node-red]
./entrypoint.sh: line 14:     7 Trace/breakpoint trap   (core dumped) /usr/local/bin/node $NODE_OPTIONS node_modules/node-red/red.js --userDir /data $FLOWS "${@}"
```

this is since a few days without changing anything. before it was running without problems.

changed logging level to debug but didnt provide more infos. and the error message dont point me anywhere :/ 

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: v3.1.0-beta.4 and latest release
- Node.js version: v16.20.1
- npm version: 8.19.4
- Platform/OS: docker
- Browser: Chrome

**Comment by wuast94** [comment]
also i did a fresh install with the beta, imported my flow and installed the missing nodes

**Comment by knolleary** [comment]
This type of issue is *typically* due to a native component of one of the extra modules you are using; the default palette of Node-RED doesn't include any native libraries.

What additional modules have you got installed?

**Comment by wuast94** [comment]
Sry for the screenshots but don't have any other option ATM.

![IMG_20230904_230714](https://github.com/node-red/node-red/assets/16965460/6fcbe183-0d0f-4f5d-b44b-b2e265fd7336)
![IMG_20230904_230744](https://github.com/node-red/node-red/assets/16965460/46df979f-421a-40a2-a845-b66389e8c48d)
![IMG_20230904_230819](https://github.com/node-red/node-red/assets/16965460/7819740e-cca0-4126-a231-b6fb2c72fb24)

**Comment by knolleary** [comment]
@wuast94 we aren't in a position to check all of those nodes for you. All I can do is offer some suggestions for you to identify which node it might be.

I would
 - check to see if any of those nodes have received an update recently
 - check for any issues reported against those nodes to see if someone else has hit a similar thing
 - look for any that include native components - that can be quite tricky to figure out without digging into the repo for each node as it is more likely they have a dependency on another npm module that has the native component.

**Comment by wuast94** [comment]
Just for Documentation:
1. I searched for native modules with: `find node_modules -type f -name "*.node" 2>/dev/null | grep -v "obj\.target" `
2. I listed the dependency tree with: `npm ls --all`
3. Looked into the dependency tree that what modules are using the native components

I had 2 matches, openapi-red and node-red-contrib-matrix-chat

Booth arent updated recently, but after removing the matrix one my errors are gone. reinstall it node red works. 

as soon as i connect to a matrix server nodered crashes again, and after removing the config from the node and it dont connect to server anymore i dont get crashes.

what makes me wonder is that this runs for over a month without problems and now breaks. sadly the node isnt maintained anymore and im not a programmer, so will have to find a way around this.

thanks for help 😊
